extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -471,6 +471,7 @@ export class ConsentUI {
    */
   createPromptIframe_(promptUISrc) {
     const iframe = this.parent_.ownerDocument.createElement('iframe');
+    iframe.setAttribute('title', this.parent_.title || 'Consent Banner');
```

Comment:
Nit: move this one row below.

Reply:
Thanks, updated with daab62b58aa12b457260647c64dda45d21bc0dd9